convert2rhel/utils.py

```diff
@@ -472,10 +472,19 @@ def download_pkg(
     output, ret_code = run_cmd_in_pty(cmd, print_output=False)
     if ret_code != 0:
         loggerinst.warning(
-            "Couldn't download the %s package using yumdownloader.\n"
-            "Output from the yumdownloader call:\n%s" % (pkg, output)
+            " you are using an env that overrides an inhibitor and"
```

Comment:
```suggestion
            "You are using an env that overrides an inhibitor and"
```

Reply:
the new code doesn't use this message anymore.